book/tests/snapshots/snapshot__run_reference_prql@examples__variables-0.prql.snap

```diff
@@ -19,16 +19,16 @@ SELECT
   title,
   country,
   AVG(salary),
-  AVG(gross_salary),
+  AVG(_expr_1),
   SUM(salary),
-  SUM(gross_salary),
-  AVG(gross_cost),
-  SUM(gross_cost) AS sum_gross_cost,
+  SUM(_expr_1),
```

Comment:
One downside of this is that DBs will often give a reasonable name for unnamed aggregations.

So we might have got `sum_gross_salary` prior, but now we get `sum__expr_1`.

(as ever, not some verdict on the overall change, and from the peanut gallery)

Reply:
Yes that is true - we basically have to replicate their naming rules and if the target database does infer some name, override it with what the RQ dictates.

This would be done here: https://github.com/prql/prql/blob/7a23912e998886c6df38870d19c9119f1dc72998/prql-compiler/src/sql/codegen.rs#L346-L352